packages/compiler-core/src/transforms/vIf.ts

```diff
@@ -26,7 +26,7 @@ import {
   MemoExpression
 } from '../ast'
 import { createCompilerError, ErrorCodes } from '../errors'
-import { processExpression } from './transformExpression'
+import { processExpression } from '@vue/compiler-core'
```

Comment:
same here and in other imports, they should reference local imports within a package